addons/sale_crm/wizard/crm_opportunity_to_quotation.py

```diff
@@ -19,13 +17,28 @@ def default_get(self, fields):
         if active_model != 'crm.lead':
             raise UserError(_('You can only apply this action from a lead.'))
 
-        active_id = self._context.get('active_id')
-        if 'lead_id' in fields and active_id:
-            result['lead_id'] = active_id
+        lead = False
+        if result.get('lead_id'):
+            lead = self.env['crm.lead'].browse(result['lead_id'])
+        elif 'lead_id' in fields and self._context.get('active_id'):
+            lead = self.env['crm.lead'].browse(self._context['active_id'])
+        if lead:
```

Comment:
```
lead = self.env['crm.lead'].browse(result.get('lead_id', 0))
if not lead and 'lead_id' in fields and self._context.get('active_id'):
    lead = self.env['crm.lead'].browse(self._context['active_id'])
if lead:
    ...
```
but maybe this is more indigest :thinking:

Reply:
Mmh, browsing 0 is risky. I agree this is a bit heavy. Meh.